src/clj/game/cards-identities.clj

```diff
@@ -74,7 +74,7 @@
                                                                 (- (get-defer-damage state side :net nil) 1)
                                                                 " more net damage")))
                                                   :effect (effect (clear-wait-prompt :runner)
-                                                                  (trash target {:cause :net})
+                                                                  (trash target {:cause :net :unpreventable true})
```

Comment:
Curious about this. Won't this make Chronos's net damage unpreventable too? Or does this part kick in already after the net damage prevention part?